Client/Frontend/BraveVPN/BraveVPN.swift

```diff
@@ -198,13 +200,25 @@ class BraveVPN {
         }
     }
     
+    /// Stores a in-memory list of vpn errors encountered during current browsing session.
+    private(set) static var errorLog = [(Date, String)]()
+    
+    /// Prints out the error to the logger and stores it in a in memory array.
+    /// This can be further used for a customer support form.
+    private static func logAndStoreError(_ message: String, printToConsole: Bool = true) {
+        if printToConsole {
+            log.error(message)
+        }
+        errorLog.append((Date(), message))
```

Comment:
Are we worried at all about this ballooning to thousands of entries and creating memory problems for older phones?

Reply:
no, but capping it is a good idea, i will do it